src/core/bpf/meson.build

```diff
@@ -65,13 +65,23 @@ bpf_o_unstripped_cmd += [
         '@OUTPUT@'
 ]
 
-bpf_o_cmd = [
-        llvm_strip,
-        '-g',
-        '@INPUT@',
-        '-o',
-        '@OUTPUT@'
-]
+if bpftool_strip
+        bpf_o_cmd = [
+                bpftool,
+                'g',
+                'o',
+                '@OUTPUT@',
+                '@INPUT@'
```

Comment:
is @INPUT@ always a single file? `bpftool gen object` allows statically linking BPF application that consists of multiple .bpf.o objects, that's it's main purpose. Stripping out DWARF info is just a by product. So I'm wondering if ability to do multi-file BPF programs was added by this PR as well or it's purely about stripping out DWARF?

Reply:
> is @input@ always a single file?

Should be at the moment.

> Stripping out DWARF info is just a by product. So I'm wondering if ability to do multi-file BPF programs was added by this PR as well or it's purely about stripping out DWARF?

I didn't make any changes for multi-file BPF program support.